src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -76,11 +76,15 @@ public DistributedApplicationBuilder(DistributedApplicationOptions options)
 
         _innerBuilder.Logging.AddFilter("Microsoft.Hosting.Lifetime", LogLevel.Warning);
         _innerBuilder.Logging.AddFilter("Microsoft.AspNetCore.Server.Kestrel", LogLevel.Error);
+        _innerBuilder.Logging.AddFilter("Aspire.Hosting.Dashboard", LogLevel.None);
```

Comment:
If the Dashboard crashes, the user doesn't get any indication of what happened, right? They just see a hang. 

https://github.com/dotnet/aspire/issues/3429

We should be at least logging errors by default.

Reply:
We can’t. I tried the playground samples and lots of other errors showed up. 

I think it could be done if we filtered to the startup category by default, but it can’t be all errors.